week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -57,8 +78,42 @@ function exerciseTwo(shopping) {
 
     The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
 **/
+
+
 function exerciseThree(books) {
   //Write your code in here
+  const holder = document.querySelector("#content");
+  const bookListHolder = document.createElement('ul');
+    holder.append(bookListHolder)
```

Comment:
Some formatting errors here - are you using this?

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Reply:
i changed all code format in prettier.